### DISCUSSION TITLE
Use Project field's color in "Insight" graphs

### DISCUSSION THREAD

**Discussion by FloEdelmann** [community/community#47798] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Last week, you [introduced](https://github.blog/changelog/2023-02-09-github-projects-feburary-9th-update/) colors for single-select fields in Projects, which is great!

However, it seems odd to me that the color is not used in the *Insight* graphs, and instead random colors are assigned:

![image](https://user-images.githubusercontent.com/202916/219722863-25494455-7ef7-4c65-beda-2788f1bb2331.png)

![image](https://user-images.githubusercontent.com/202916/219723044-8e68d03e-8e29-4f20-877d-cec3e1f5662f.png)

Project from where the screenshots are taken: https://github.com/users/FloEdelmann/projects/2/insights/1

**Comment by ncb-hydrema** [comment]
I definitely agree. A lot of times, the field coloring is used for indicating some kind of attribute (good vs. bad, big vs. small etc.). Having different colours on the charts is somewhat confusing. Somehow field value "Critical" mostly shows up as green for me :)
